.github/workflows/build.yml

```diff
@@ -37,13 +57,10 @@ jobs:
 
       - name: Test with Maven
         run: mvn test
-      
-      - name: Install with Maven
-        run: mvn install
```

Comment:
The "Build with Maven" step on line 55 not only installs, but I expect it also runs the unit tests.

(Suggestion is to add a `-DskipTests` to suppress the tests during the build.)

Reply:
I always make this mistake. I want a build and running of tests to be separate steps. I want to be able to see the difference between not compilable code and failing tests..
I've added `-DskipTests` in this workflow.